exercises/prime-factors/prime_factors_test.exs

```diff
@@ -62,4 +62,16 @@ defmodule PrimeFactorsTest do
   test "93819012551" do
     assert PrimeFactors.factors_for(93819012551) == [11, 9539, 894119]
   end
+
+  @tag :optional
+  @tag timeout: 120000
+  test "performance optimizations" do
+    %{minute: original_minutes, second: original_seconds} = DateTime.utc_now
+    PrimeFactors.factors_for(10000000055) == [5, 2000000011]
```

Comment:
@dalexj I ended up splitting it up into two tests - that made the most sense to me. I also changed the assertion as to how fast that function returns, since I saw on the second iteration from @sunboshan that he got it to finish in 2.4ms, so I figured under 2 seconds would be a safe max to make that test pass.